packages/web-components/src/components/va-input-telephone/va-input-telephone.tsx

```diff
@@ -0,0 +1,247 @@
+import {
+  Component,
+  Element,
+  Host,
+  Prop,
+  h,
+  Listen,
+  State,
+  EventEmitter,
+  Event
+} from '@stencil/core';
+
+import { AsYouType, isValidPhoneNumber, CountryCode, getExampleNumber, getCountries, getCountryCallingCode } from 'libphonenumber-js/min'; 
+import examples from 'libphonenumber-js/examples.mobile.json';
+import classNames from 'classnames';
+
+/**
+ * @componentName Input Telephone
+ * @maturityCategory caution
+ * @maturityLevel proposal
+ * @guidanceHref form/input-telephone
+ * @translations English
+ */
+
+@Component({
+  tag: 'va-input-telephone',
+  styleUrl: 'va-input-telephone.scss',
+  shadow: true,
+})
+export class VaInternationalTelephone {
+  @Element() el: HTMLElement;
+  textInputRef: HTMLVaTextInputElement;
+  DEFAULT_COUNTRY: CountryCode = 'US';
+
+  /**
+   * The telephone contact information
+   */
+  @Prop() contact?: string;
+
+  /**
+   * The 2 letter ISO country code for a country
+   */
+  @Prop({ reflect: true, mutable: true }) country?: CountryCode = this.DEFAULT_COUNTRY;
+
+
+  /**
+   * Header text for the component
+   */
+  @Prop() header?: string = 'Primary phone number';
+
+  /**
+   * Hint string text
+   */
+  @Prop() hint?: string;
+
+  /**
+   * The error for the component
+   */
+  @Prop({ reflect: true, mutable: true }) error?: string = '';
+
+  /**
+   * Whether the country select should be included. Set to true to exclude it.
+   */
+  @Prop() noCountry?: boolean = false;
+
+  /**
+   * The event emitted when the contact changes
+   */
+  @Event() vaContact: EventEmitter;
+
+  /**
+   * The event emitted when the country code changes
+   */
+  @Event() vaCountryCode: EventEmitter
+
+  /**
+   * The contact formatted according to the selected country
+   */
+  @State() formattedContact: string = '';
+
+  /**
+   * List of sorted countries used in the country combo-box
+   */
+  @State() countries: string[] = null;
+
+  /**
+   * Error for the country combo-box
+   */
+  @State() countryError: string = '';
+
+  /**
+   * Error for the contact input
+   */
+  @State() contactError: string = '';
+
+  resetErrors() {
+    this.countryError = '';
+    this.contactError = '';
+    this.error = '';
+  }
+
+  updateContact(event: InputEvent) {
+    const target = event.target as HTMLInputElement;
+    const { value } = target;
+    this.formattedContact = value ? this.formatContact(value) : '';
+  }
+
+  // get an error message for a country that includes the template of a valid phone number for that country
+  getErrorMessageForCountry() {
+    let example = getExampleNumber(this.country, examples).format('NATIONAL');
```

Comment:
Just for my own understanding, the `examples` import from libphonenumber-js is for showing an example of a generic country phone number in the error message in case the entered number doesn't pass validation? Is there a reason it's called "mobile"?  `'libphonenumber-js/examples.mobile.json';`

Reply:
@jamigibbs This is a good point. Basically, the example chosen will be a mobile number. If we use a larger dataset then it might be possible to get a mobile and landline for each country (sometimes the format differs); but it seems like it would be a little clumsy to display both, it would not always be an issue, and the bundle would increase, so I went with mobile. Thoughts?

Reply:
That makes sense. We can just put a pin in this for now and revisit if we find there's confusion with the formatting once it's put in front of users.